src/dispatch/plugins/dispatch_slack/case/interactive.py

```diff
@@ -0,0 +1,493 @@
+
+    case_type = None
+    if form_data.get(DefaultBlockIds.case_type_select):
+        case_type = {"name": form_data[DefaultBlockIds.case_type_select]["name"]}
+
+    case_in = CaseUpdate(
+        title=form_data[DefaultBlockIds.title_input],
+        description=form_data[DefaultBlockIds.description_input],
+        resolution=form_data[DefaultBlockIds.resolution_input],
+        status=CaseStatus.closed,
+        visibility=case.visibility,
+        case_priority=case_priority,
+        case_type=case_type,
+    )
+
+    case = case_service.update(db_session=db_session, case=case, case_in=case_in, current_user=user)
+    blocks = create_case_message(case=case, channel_id=context["subject"].channel_id)
+    await client.chat_update(
+        blocks=blocks, ts=case.conversation.thread_id, channel=case.conversation.channel_id
+    )
+
+
+@app.shortcut(CaseShortcutCallbacks.report, middleware=[db_middleware, shortcut_context_middleware])
+async def report_issue(ack, shortcut, body, context, db_session, client):
+    await ack()
+    initial_description = None
+    if body.get("message"):
+        permalink = (
+            await client.chat_getPermalink(
+                channel=context["subject"].channel_id, message_ts=body["message"]["ts"]
+            )
+        )["permalink"]
+        initial_description = f"{body['message']['text']}\n\n{permalink}"
+
+    blocks = [
+        Context(
+            elements=[
+                MarkdownText(text="Fill the following form out to the best of your abilities.")
+            ]
+        ),
+        title_input(),
+        description_input(initial_value=initial_description),
+        project_select(
+            db_session=db_session,
+            action_id=CaseReportActions.project_select,
+            dispatch_action=True,
+        ),
+    ]
+
+    modal = Modal(
+        title="Report Issue",
+        blocks=blocks,
+        submit="Report",
+        close="Close",
+        callback_id=CaseReportActions.submit,
+        private_metadata=context["subject"].json(),
+    ).build()
+    await client.views_open(trigger_id=shortcut["trigger_id"], view=modal)
+
+
+@app.action(CaseReportActions.project_select, middleware=[db_middleware, action_context_middleware])
+async def handle_report_project_select_action(ack, body, db_session, context, client):
+    await ack()
+    values = body["view"]["state"]["values"]
+
+    project_id = values[DefaultBlockIds.project_select][CaseReportActions.project_select][
+        "selected_option"
+    ]["value"]
+
+    project = project_service.get(
+        db_session=db_session,
+        project_id=project_id,
+    )
+
+    blocks = [
+        Context(elements=[MarkdownText(text="Accept the defaults or adjust as needed.")]),
+        title_input(),
+        description_input(),
+        project_select(
+            db_session=db_session,
+            initial_option={"text": project.name, "value": project.id},
+            action_id=CaseEscalateActions.project_select,
```

Comment:
```suggestion
            action_id=CaseReportActions.project_select,
```

I think this is what we want. Selecting `Project` twice switches to an escalate modal.